client/src/components/ThemeSwitch.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Switch from 'react-switch';
+
+function Themeswitch() {
+  const [isToggled, setToggled] = React.useState(false);
+
+  const handleChange = () => setToggled(!isToggled);
+
+  return (
+    <Switch
+      onChange={handleChange}
+      checked={isToggled}
+      aria-labelledby="neat-label"
```

Comment:
nice. accessibility :-) you could actually name this "theme-switch-toggle" or something. this is important for screen readers

Reply:
true!